### ISSUE TITLE
How to query current MRV2 version via Python

### ISSUE THREAD

**Issue by BigRoy** [ggarra13/mrv2#131]
How do I query the current version of MRV2 via the Python API?

I would've expected a `cmd.getVersion()` function to exist.

**Comment by ggarra13** [comment]
You are right, there's no such function.  I added it to the beta branch.  I am waiting for the beta branch to compile fine on all platforms and then I'll merge it into main.

Are you compiling from source, or downloading the betas from sourceforge.net btw?

**Comment by BigRoy** [comment]
I'm currently downloading the releases each time and just reporting issues as I go. (Sorry for the spam of issues, hehe)
Haven't had the need to compile myself for my current development.

**Comment by ggarra13** [comment]
Added in HEAD.
